gatsby-node.js

```diff
@@ -67,12 +69,20 @@ exports.createPages = ({ actions, graphql }) => {
       })
     })
 
-    // categories.forEach(edge => {
-    //   createPage({
-    //     path: `category/${categoryName}`,
-    //     component: path.resolve(`src/templates`)
-    //   })
-    // })
+    const allCategories = result.data.allGoogleSheetValue.edges 
+
+    allCategories.forEach(edge => {
+      const { Category_1 } = edge.node
+      const categoryPath = `/category/${_.kebabCase(Category_1)}/`
+
+      createPage({
+        path: categoryPath,
+        component: path.resolve(`src/templates/singleCategory.js`),
```

Comment:
Just a typo but when I run this locally I get the error below. So the s just needs to be capitalised in SingleCategory to fix.

 "warn Your site's "gatsby-node.js" created a page with a component path that
doesn't match the casing of the actual file. This may work locally, but will
break on systems which are case-sensitive, e.g. most CI/CD pipelines."

Reply:
Thanks, I've amended!